### ISSUE TITLE
"offset" calculation issue

### ISSUE THREAD

**Issue by AlexeySvidentsov** [scroll-out/scroll-out#58]
I'm trying to make elements visible when they appears in screen for 150px;
`ScrollOut( {
	offset: function() {
		return window.innerHeight - 150;
	}
}`
And according documentation: 

> offset = The targets become visible when they reach this distance in pixels from the top of the screen. Setting this option overrides all other collision detection

So as i understand correctly it should take target offset related to the screen and compare it to the specified value (in my case window.InnerHeight - 150px)

But digging in the code i have seen that it compares scroll value instead of actual target offset.
Is there something i've missed out?

**Comment by notoriousb1t** [comment]
You are correct, it is based the offset between the top of the scrolling content and the scrolling container. The documentation is incorrect and needs to be updated (I'll take this as an action item and update it). This feature is intended for sticky nav bars, so that is why the documentation is written incorrectly that way.

It seems like you have located the correct place to put that logic: https://github.com/scroll-out/scroll-out/blob/master/src/main.ts#L105. I am open for PRs to provide this functionality since it seems pretty straightforward and instantly useful. This can be exposed as a new property perhaps:

```ts
ScrollOut({
  visiblePixels: 150 // Number of pixels the target must overlap with the viewport.
})
```
